modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,29 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== API
+
+The Comments plugin exposes an API, `editor.plugins.tinycomments.getEventLog()`, which returns a log of all Comment-related events, including when:
+
+* A new comment is added (and who added it).
+* A comment is edited (and who edited it).
+* A reply to a comment is added (and who added the reply).
+* A comment is resolved (and who resolved the comment).
+* A comment is deleted (and who deleted the comment).
+
+The event log can be retrieved either in full or with the `after` option, which restricts the returned list to Comment events after the time-stamp date, both shown in the following:
+
+NOTE: This feature is only available for {productname} 6.1 and later.
```

Comment:
```suggestion
NOTE: This feature is only available for {productname} 6.1 and later.

The event log can be retrieved either in full or with the `after` option, which restricts the returned list to Comment events after the time-stamp date, both shown in the following:
```
I think reversing the paragraph order here puts the admonition in a less disruptive spot.

The admonition is now immediately after the feature listing (which is what requires 6.1 or later) and the ‘here’s the sample code’ sentence is now immediately adjacent the sample code.